library/tiqr/Tiqr/Message/Exception/SendFailure.php

```diff
@@ -19,7 +19,7 @@
 
 
 /** @internal includes */
-require_once 'Message/Exception.php';
+//require_once 'Message/Exception.php';
```

Comment:
Are these includes needed at all? Couldn't we rely on autoloading? These includes won't work as is (file not found).
Same goes for AuthFailure and InvalidAddress exceptions